Update what-is-the-default-directory-that-zap-uses.md (#638)
zaproxy/zaproxy-website: site/content/faq/what-is-the-default-directory-that-zap-uses.md

`~/Library/Application Support/ZAP`

##  Weekly releases and development builds

> Note that for weekly releases `_D` is appended to the above directory. 

Reviewer: IMO this should be tweaked too, to keep it clear that both weekly and development builds use the directory.

Reviewer 2: Ya it could just be a unified statement, like:
> Note that for weekly releases and dev builds `_D` is appended to the above mentioned directory name.

Author: ```suggestion
Note that `_D` is appended to the above directory. E.g. in Linux the zap.log file will be saved in ~/.ZAP_D
```

Author: ```suggestion
Note that `_D` is appended to the above directory. E.g. in Linux the home directory is ~/.ZAP_D
```

> Note that for weekly releases `_D` is appended to the above directory. 

Reviewer 2: ```suggestion
Note that for both weekly releases and dev builds `_D` is appended to the above mentioned directory name. For example on Linux the zap.log file will be saved in `~/.ZAP_D/`.
```

Reviewer 2: Oh it didn't add it to the existing thread :cry: Silly GitHub interface.

Author: great

Reviewer: Not sure about the zap.log reference, that's not mentioned anywhere else in this FAQ and we already have one for that.

Reviewer 2: ```suggestion
Note that for both weekly releases and dev builds `_D` is appended to the above mentioned directory name. For example on Linux `~/.ZAP_D/` will be used.
```

Is that better?

> Note that for both weekly releases and dev builds `_D` is appended to the above mentioned directory name. For example on Linux the zap.log file will be saved in `~/.ZAP_D/`.

Reviewer 2: ```suggestion
Note that for both weekly releases and dev builds `_D` is appended to the above mentioned directory name. For example on Linux the home directory is `~/.ZAP_D`.
```

> Note that for both weekly releases and dev builds `_D` is appended to the above mentioned directory name. For example on Linux the zap.log file will be saved in `~/.ZAP_D/`.

Reviewer 2: ```suggestion
Note that for both weekly releases and dev builds `_D` is appended to the above mentioned directory name. For example on Linux the home directory is `~/.ZAP_D/`.
```

Reviewer 2: Per the latest messages yesterday.

Reviewer 3: +1 if the user uses the `-dir` option then that will be used for the log files instead of the home dir...

> Note that for both weekly releases and dev builds `_D` is appended to the above mentioned directory name. For example on Linux the home directory is `~/.ZAP_D/`. If the user specified -dir option then that parameter will be used as home directory  

Author: yes you are right, its going to be duplicate
